org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/parameter-process.css.dsp

```diff
@@ -8,6 +8,9 @@ when detect side effect, fix to only apply for parameter window*/
 .main-parameter-layout {
 	height: 100%;
 }
+.process-modal-dialog {
+	width: 600px;
+}
```

Comment:
need to add @media screen and max-width to cater for mobile screen with width < 600px